cache-config/t3c-apply/config/config.go

```diff
@@ -681,8 +681,7 @@ func getOSSvcManagement() SvcManagement {
 		_svcManager = SystemD
 	} else if isCommandAvailable(Service) {
 		_svcManager = SystemV
-	}
-	if !isCommandAvailable(Chkconfig) {
+	} else {
```

Comment:
Since systemV needs chkconfig maybe this would be better. 
`if _svcManager == SystemV && !isCommandAvailable(Chkconfig) {return Unknown}`